api/src/organizational-entities/domain/errors.js

```diff
@@ -20,6 +20,14 @@ class ArchiveCertificationCentersInBatchError extends DomainError {
   }
 }
 
+class ArchiveOrganizationsInBatchError extends DomainError {
+  constructor({ code = 'ARCHIVE_ORGANIZATIONS_IN_BATCH_ERROR', meta } = {}) {
+    super();
+    this.code = code;
+    this.meta = meta;
+  }
+}
```

Comment:
Il ne faut pas _« mettre un code dans le constructeur qui incite à le changer »_ cf.
https://github.com/1024pix/pix/blob/dev/docs/errors.md, aussi ci-dessous une autre implémentation possible de la classe d'erreur suivant une forme qui est déjà utilisée pour la `TagNotFoundError` : 

```suggestion
class ArchiveOrganizationsInBatchError extends DomainError {
  constructor(meta) {
    super();
    if (meta) this.meta = meta;
  }
}
```